java/Oleksii-Riabov/src/com/zoolatech/lecture3/tasks/_2/NumberValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class NumberValidator extends AccountValidator {
+    String fieldName;
+
+    public NumberValidator(String fieldName) {
+        this.fieldName = fieldName;
+    }
+
+    @Override
+    public boolean isValid(UserAccount userAccount) {
+        switch (fieldName) {
+            case "userId" -> {
+                System.out.println("Validating that serId field: " + userAccount.userId() + " is bigger than 0");
+                return true;
```

Comment:
return true; can be moved out of the switch statement to the end of the method - this will allow to remove duplicated code

Reply:
This issue is solved.